go.mod

```diff
@@ -95,3 +103,5 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.7.0 // indirect
 	sigs.k8s.io/yaml v1.4.0 // indirect
 )
+
+tool github.com/oapi-codegen/oapi-codegen/v2/cmd/oapi-codegen
```

Comment:
I expect you also want to change the Makefile to run `go tool` rather than `go install`?  You win bonus points for applying this change to `core,identity,region,kubernetes` too to maintain consistency 😉